test/middleware_test.rb

```diff
@@ -219,7 +219,7 @@ def test_patch_called_with_alias_method_instrument
     Object.stub_const(:Dalli, Module) do
       ::Dalli.stub_const(:Client) do
         mock = Minitest::Mock.new
-        mock.expect :call, nil, [Dalli::Client, Array, :memcache, { instrument: :alias_method }]
+        mock.expect :call, nil, [Dalli::Client, Array, :memcache], instrument: :alias_method
```

Comment:
I don't 100% know my way around but if we reference the [other lines](https://github.com/discourse/prometheus_exporter/pull/330/files#diff-505ed55d84d921ba53ef45426fc4166a933138c6fe0f8bf7f31df1703cff7698L209-L211) this fixes the build